account.go

```diff
@@ -65,17 +92,19 @@ func RecordToAccount(rec Record, genesisTime time.Time) (Account, error) {
 		}, nil
 	}
 
+	// calculate dust, which represents an uregen-integral remainder, represented as an apd.Decimal of regen
```

Comment:
```suggestion
	// calculate dust, which represents an uregen-integral remainder, represented as a decimal value of regen
```
Removing apd.Decimal from comment